sdk/test-utils/perf/package.json

```diff
@@ -22,7 +22,7 @@
     "integration-test:node": "node dist-esm/test/index.spec.js",
     "integration-test": "npm run integration-test:node && npm run integration-test:browser",
     "lint:fix": "eslint --no-eslintrc -c ../../.eslintrc.internal.json package.json src test --ext .ts --fix --fix-type [problem,suggestion]",
-    "lint": "eslint --no-eslintrc -c ../../.eslintrc.internal.json package.json src test --ext .ts -f html -o perf-lintReport.html || exit 0",
+    "lint": "eslint --no-eslintrc -c ../../.eslintrc.internal.json package.json src test --ext .ts -f html -o perf-lintReport.html",
```

Comment:
why are we still generating the report?

Reply:
Just assumed that all that needed to be fixed was the default exit code. Why should we not be generating the report in this case?

Reply:
Hmm, maybe we don't need that because the command already outputs the errors and warnings to the console?
Why do we not have this output file generation in other SDKs, @deyaaeldeen?

Reply:
@HarshaNalluru it is easier to inspect a report printed in the console than in an html file because it can be painful to setup a display server to open the browser in a dev box (if you use one).

Reply:
Ah yeah, I see, that’s a good reason. So it appears that eslint doesn’t output to the console when you ask it to create an HTML report. I assumed it did when I made my original comment, so was wondering what the downside was 😊. I’ll simplify as suggested.